IPtProxy.go/controller.go

```diff
@@ -287,13 +288,20 @@ func (c *Controller) Start(methodName string, proxy string) {
 	case "snowflake":
 		iceServers := strings.Split(strings.TrimSpace(c.SnowflakeIceServers), ",")
 		frontDomains := strings.Split(strings.TrimSpace(c.SnowflakeFrontDomains), ",")
+
+		maxPeers := c.SnowflakeMaxPeers
+		if maxPeers < 1 {
+			maxPeers = 1
+		}
+
```

Comment:
This check isn't necessary to do here, since we do it in the Snowflake library as well when we create the transport with the `Config`. But, I have a followup commit to remove `snowflake.go` that will change this anyway.